styles/common/button.scss

```diff
@@ -1,8 +1,6 @@
 @mixin button {
-  height: 25vmin;
-  width: 25vmin;
-  min-width: 130px;
-  min-height: 130px;
+  height: 28vmin;
+  width: 28vmin;
```

Comment:
iPhone SE 2016: labels are out of buttons (both vertically and horizontally)
large screens: buttons are not in one line